Sources/ArcGISToolkit/Components/Offline/PreplannedMetadataView.swift

```diff
@@ -0,0 +1,125 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct PreplannedMetadataView: View {
+    /// The view model for the preplanned map.
+    @ObservedObject var model: PreplannedMapModel
+    
+    /// The action to dismiss the view.
+    @Environment(\.dismiss) private var dismiss: DismissAction
+    
+    public var body: some View {
+        Form {
+            Section {
+                if let image = model.preplannedMapArea.thumbnail {
+                    HStack {
+                        Spacer()
+                        LoadableImageView(loadableImage: image)
+                            .clipShape(.rect(cornerRadius: 10))
+                            .padding(.vertical, 10)
+                            .shadow(radius: 10)
```

Comment:
```suggestion
                            .shadow(radius: 5)
```
The shadow radius can be reduced:
<img src="https://github.com/user-attachments/assets/2a70824b-f89f-4d27-a3a6-2b6a698d9165" width="300">

Is there a way to make the image larger without compromising image quality?

Reply:
I have changed the max width and height to 1.5x of the thumbnail's native resolution. Let me know what you think.

Reply:
Looks good :+1:

<img src="https://github.com/user-attachments/assets/8074d104-1bab-4c41-aee9-8fec2104575d" width="300">